First take on Application Names section

ernesto-jimenez opened the pull request:
Hi @marcoscaceres,

Here's a first take on the section names.

This is not intended to be merged yet but to get your feedback.

It's my first time editing, so it would be great to have your comments.

Cheers,
Ernesto

ernesto-jimenez committed:
First take on Application Names section

marcoscaceres commented on sections/names.html:
```diff
@@ -1,54 +1,114 @@
+</ol>
+
+<p>iOS and Android developers also need to define a listings name during the
+application submision process to the store which is used in the store listings.
```
typo: submision

marcoscaceres commented on sections/names.html:
```diff
@@ -1,54 +1,114 @@
+
+<p>Since iOS6, apple supports it's own version of "application-name" metadata
+called "apple-mobile-web-app-title". However, it doesn't use "application-name"
+(TODO: VERIFY)</p>
```
TODO: we should note the fallback to using the title element.

marcoscaceres commented on sections/names.html:
```diff
@@ -1,54 +1,114 @@
+
+<h4>Chrome Mobile</h4>
+
+<p>TODO: CHECK</p>
```
They removed installed web apps support in the latest release :(

marcoscaceres commented on sections/names.html:
```diff
@@ -1,54 +1,114 @@
+<ol>
+  <li><strong>"application-name"</strong>: used in 1,571 sites.</li>
+  <li><strong>"apple-mobile-web-app-title"</strong>: used in 244 sites. 50 of
+  those sites used "application-name" too.</li>
```
Nice find! :)

marcoscaceres commented:
This is looking good!

ernesto-jimenez committed:
Updated browser status and fixed typo

ernesto-jimenez commented:
Fixed the typo and updated the browser status:
- Clarify Safari Mobile falls-back to title and confirmed it works as stated
- Confirmed Chrome Mobile and Firefox for Android only use the title tag

marcoscaceres commented:
@ernesto-jimenez you ready for me to merge? Or you want me to keep reviewing?

ernesto-jimenez commented:
I want to double-check the ReSpec output because I think it was doing some weird things with the H4 headers.

Apart from that, I think it should be OK to merge if this looks good enough to you. We have some reviews left on the document, so it'll probably be OK.

marcoscaceres commented:
Reviewed. All looks good to me!
